recommend to change the NAME to be in UPPER CASE format by force and new vehicle format.

diniemasjuki opened the pull request:
make it more formal and standardize format in uppercase format for names of:
company
customer
supplier
Lead

diniemasjuki committed:
Update vehicle.json

diniemasjuki committed:
Update vehicle.json

diniemasjuki committed:
Update vehicle.json

diniemasjuki committed:
Update vehicle.json

diniemasjuki committed:
Update vehicle.json

diniemasjuki committed:
Update vehicle.json

diniemasjuki committed:
Update vehicle.json

diniemasjuki committed:
Update vehicle.json

diniemasjuki committed:
Update vehicle.json

diniemasjuki committed:
Update vehicle.json

diniemasjuki committed:
Update vehicle.json

diniemasjuki committed:
Update vehicle.json

diniemasjuki committed:
Update vehicle.json

diniemasjuki committed:
Update vehicle.json

diniemasjuki committed:
Update vehicle.py

diniemasjuki committed:
Update vehicle.py

diniemasjuki committed:
Update vehicle.py

diniemasjuki committed:
Update vehicle.json

diniemasjuki committed:
Update vehicle.py

diniemasjuki committed:
Update vehicle.json

diniemasjuki committed:
Update vehicle.json

diniemasjuki committed:
Update vehicle.js

diniemasjuki committed:
Update vehicle.js

diniemasjuki committed:
Update item.json

diniemasjuki committed:
Update item.json

diniemasjuki committed:
Update item.json

diniemasjuki committed:
Update item.py

diniemasjuki committed:
Update item.py

diniemasjuki committed:
Update item.py

diniemasjuki committed:
Update item.py

diniemasjuki committed:
Update item.py

diniemasjuki committed:
Update item.py

diniemasjuki committed:
Update item.py

diniemasjuki committed:
Update item.py

diniemasjuki committed:
Update item.json

diniemasjuki committed:
Update item.json

diniemasjuki committed:
Update item.json

diniemasjuki committed:
Update item.json

diniemasjuki committed:
Update item.json

diniemasjuki committed:
Update item.py

diniemasjuki committed:
Update item.py

diniemasjuki committed:
Update item.py

diniemasjuki committed:
Update item.py

diniemasjuki committed:
Update item.py

diniemasjuki committed:
Update item.py

diniemasjuki committed:
Update item.py

diniemasjuki committed:
Update item.py

diniemasjuki committed:
Update item.py

diniemasjuki committed:
Update item.py

diniemasjuki committed:
Merge branch 'frappe:develop' into develop

diniemasjuki committed:
Merge branch 'frappe:develop' into develop

diniemasjuki committed:
Merge branch 'frappe:develop' into develop

diniemasjuki committed:
Merge branch 'frappe:develop' into develop

diniemasjuki committed:
Update stock_settings.json

make naming series as default

diniemasjuki committed:
Update stock_settings.json

Setting Default Stock UOM as Unit

diniemasjuki committed:
Update selling_settings.json

Set default  customer default as naming series

diniemasjuki committed:
Update buying_settings.json

Supplier Naming by Series

diniemasjuki committed:
Update bank_transaction.json

diniemasjuki committed:
Update cashier_closing.json

Fix default naming series already used in POS Closing Entry

diniemasjuki committed:
Update company.py

force company title to be uppercase

diniemasjuki committed:
Merge branch 'frappe:develop' into wp-edit

diniemasjuki committed:
Update stock_settings.json

fix the update

diniemasjuki committed:
Update company.py

diniemasjuki committed:
Update company_setup.py

make company name uppercase format

diniemasjuki committed:
Update employee.py

make all name upper case

diniemasjuki committed:
Update company.py

trying to make company name upper cased

diniemasjuki committed:
Update company.py

fix upper case

diniemasjuki committed:
Update company.py

upper case the abbr

diniemasjuki committed:
Update supplier.py

make supplier name upper case

diniemasjuki committed:
Update customer.py

make customer names upper case format automatically

diniemasjuki committed:
Update lead.py

make the lead name all in uppercase format

diniemasjuki committed:
Update customer.py

fix uppercase format

diniemasjuki committed:
Update customer.py

fix uppercase missing

diniemasjuki committed:
Update customer.py

fix uppercase

diniemasjuki committed:
Update customer.py

fix syntax error

diniemasjuki committed:
Update customer.py

fix missing variable handler

diniemasjuki committed:
Update customer.py

fix syntax error

diniemasjuki committed:
Update lead.py

fix variable missing

diniemasjuki committed:
Update lead.py

fix variable missing

diniemasjuki committed:
Update prospect.py

make company name upper case format

diniemasjuki committed:
Merge branch 'frappe:develop' into wp-edit

diniemasjuki committed:
Merge branch 'frappe:develop' into wp-edit

diniemasjuki committed:
Update account.py

fix error code:

### App Versions
```
{
	"agriculture": "0.0.1",
	"autoservice": "0.0.1",
	"beam": "15.4.0",
	"builder": "2.0.0-dev",
	"dialog_upi": "0.0.1",
	"ecommerce_integrations": "1.17.0",
	"education": "15.5.0",
	"erpnext": "16.0.0-dev",
	"erpnext_telegram_integration": "1.3.0",
	"frappe": "16.0.0-dev",
	"frappe_qrcode": "0.0.1",
	"frappe_whatsapp": "1.0.7",
	"healthcare": "16.0.0-dev",
	"hrms": "16.0.0-dev",
	"insights": "3.0.0",
	"library_management": "1.0.0",
	"non_profit": "0.0.1",
	"payments": "0.0.1",
	"pibicut": "0.0.1",
	"print_designer": "1.0.0-dev",
	"webshop": "0.0.1"
}
```
### Route
```
Form/Account/TnGO eWallet - OMG
```
### Traceback
```
Traceback (most recent call last):
  File "apps/frappe/frappe/app.py", line 116, in application
    response = frappe.api.handle(request)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "apps/frappe/frappe/api/__init__.py", line 49, in handle
    data = endpoint(**arguments)
           ^^^^^^^^^^^^^^^^^^^^^
  File "apps/frappe/frappe/api/v1.py", line 36, in handle_rpc_call
    return frappe.handler.handle()
           ^^^^^^^^^^^^^^^^^^^^^^^
  File "apps/frappe/frappe/handler.py", line 51, in handle
    data = execute_cmd(cmd)
           ^^^^^^^^^^^^^^^^
  File "apps/frappe/frappe/handler.py", line 87, in execute_cmd
    return frappe.call(method, **frappe.form_dict)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "apps/frappe/frappe/__init__.py", line 1715, in call
    return fn(*args, **newargs)
           ^^^^^^^^^^^^^^^^^^^^
  File "apps/frappe/frappe/utils/typing_validations.py", line 32, in wrapper
    return func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^
  File "apps/erpnext/erpnext/accounts/doctype/account/account.py", line 482, in update_account_number
    _ensure_idle_system()
  File "apps/erpnext/erpnext/accounts/doctype/account/account.py", line 615, in _ensure_idle_system
    if last_gl_update > add_to_date(None, minutes=-5):
       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
TypeError: '>' not supported between instances of 'NoneType' and 'datetime.datetime'

```
### Request Data
```
{
	"type": "POST",
	"args": {
		"account_name": "TnGO Merchant",
		"name": "TnGO eWallet - OMG"
	},
	"headers": {},
	"error_handlers": {},
	"url": "/api/method/erpnext.accounts.doctype.account.account.update_account_number",
	"request_id": null
}
```
### Response Data
```
{
	"exception": "TypeError: '>' not supported between instances of 'NoneType' and 'datetime.datetime'",
	"exc_type": "TypeError",
	"_exc_source": "erpnext (app)"
}
```

diniemasjuki committed:
Update account.py

fix error
TypeError: '>' not supported between instances of 'NoneType' and 'datetime.datetime'

agritheory commented:
I don't personally want this feature. I think the convention of all-caps is unusual and less legible than title case. Also, it's available entirely as a customization API, so you can make this way if you want it to be this way.

diniemasjuki committed:
Update account.py

attempt fix for error:
  File "apps/erpnext/erpnext/accounts/doctype/account/account.py", line 616, in _ensure_idle_system
TypeError: '>' not supported between instances of 'NoneType' and 'datetime.datetime'

diniemasjuki commented:
> I don't personally want this feature. I think the convention of all-caps is unusual and less legible than title case. Also, it's available entirely as a customization API, so you can make this way if you want it to be this way.

thanks for the feedback sir,
